libdnf/module/ModulePackageContainer.cpp

```diff
@@ -1366,6 +1425,31 @@ void ModulePackageContainer::Impl::ModulePersistor::rollback(void)
     }
 }
 
+
+
+
+std::vector<ModulePackageContainer::stream_change_t>
```

Comment:
Nitpicking, but could you maintain a consistent number of linebreaks? :slightly_smiling_face: